config/config.yaml

```diff
@@ -32,6 +32,8 @@ s3Config:
 
 tracingConfig:
   zipkinEndpoint: ${ZIPKIN_TRACING_ENDPOINT:-http://localhost:9411/api/v2/spans}
+  commonTags:
+    clusterName: ${KALDB_CLUSTER_NAME:-kaldb_local}
```

Comment:
add env as a tag also.

Reply:
Done.

Reply:
Added the env, but left off the component for now. An individual node can participate in one or more roles, so mapping a single key/value to a comma separated list isn't tremendously useful.